example_cases/BARC/gui.yaml

```diff
@@ -0,0 +1,57 @@
+sliders:
```

Comment:
@AudreySaltzman feelings on including a BARC example? Happy to delete it if ya'll don't feel ready to have one out there.